uavcan/node/port/7510.List.0.1.uavcan

```diff
@@ -0,0 +1,22 @@
+# A list of ports that this node is using:
+# - Subjects published by this node (whether periodically or ad-hoc).
+# - Subjects that this node is subscribed to (a datalogger or a debugger would typically subscribe to all subjects).
+# - RPC services consumed by this node (i.e., service clients).
+# - RPC services provided by this node (i.e., service servers).
+#
+# All nodes should implement this capability to provide network introspection and diagnostic capabilities.
+# This message should be published using the fixed subject-ID as follows:
+# - At the OPTIONAL priority level at least every MAX_PUBLICATION_PERIOD seconds.
+# - At the OPTIONAL or SLOW priority level within MAX_PUBLICATION_PERIOD after the port configuration is changed.
+#
+# pragma: no-bit-length-limit
+
+uint8 MAX_PUBLICATION_PERIOD = 10  # [seconds]
```

Comment:
what if we suggested a register value for this where 0xFF disables the broadcast, 0 means "as fast as possible", and other values allow this to be adjustable. This would be useful for debugging where you want to up the broadcast rate so you don't have tow wait around for 10 seconds to observe changes in a tool like Yukon.

Reply:
![image](https://user-images.githubusercontent.com/3298404/97359516-8a116b80-18a5-11eb-973e-d4d22cd7b62a.png)

The reasoning here is that a system that can easily implement a behavior like that (a low-SIL system, likely) is allowed to publish a new message on change immediately, so in this case, you won't have to wait around. Also, subscription configurations are generally not changed often, even in the conventional ROS/DDS world (this is my humble empirical observation). At the same time, high-SIL systems are allowed to opt-out from the variable-rate publishing (which may be hard to reconcile with hard real-time scheduling) while staying conformant.